src/Dialog/Dialog.tsx

```diff
@@ -375,7 +375,7 @@ const Footer = styled.div<SxProp>`
   flex-shrink: 0;
 
   button {
```

Comment:
We can get rid of this whole margin rule and just use `gap: ${get('space.2')};` on the parent but that’s a bigger change and I’m not sure if we’re confidently using `gap` everywhere yet.

Reply:
A quick glance says `gap` is well supported so we can use it if you'd like to.